src/content/developers/docs/data-and-analytics/block-explorers/index.md

```diff
@@ -25,46 +24,33 @@ You should understand the basic concepts of Ethereum so you can make sense of th
 
 ## Data {#data}
 
-Ethereum is transparent by design so everything is verifiable. Block explorers provide an interface for getting this information. And this is for both the main Ethereum network and the testnets, should you need that data.
+Ethereum is transparent by design so everything is verifiable. Block explorers provide an interface for getting this information. And this is for both the main Ethereum network and the testnets, should you need that data. Data is divided into execution data and consensus data. The execution data refers to the transactions that have been executed in a specific block. The consensus data refers to the blocks themselves and the validators who proposed them.
 
 Here's a summary of the types of data you can get from a block explorer.
 
-### Blocks {#blocks}
+### Execution data {#execution-data}
 
-New blocks are added to Ethereum every ~12 seconds (this can fluctuate) so there's a near-constant stream of data that gets added to block explorers. Blocks contain a lot of important data that you may find useful:
+New blocks are added to Ethereum every 12 seconds (unless a block proposer misses its cue) so there's a near-constant stream of data that gets added to block explorers. Blocks contain a lot of important data that you may find useful:
```

Comment:
```suggestion
New blocks are added to Ethereum every 12 seconds (unless a block proposer misses its cue), so a near-constant stream of data gets added to block explorers. Blocks contain a lot of important data that you may find useful:
```

Would it still be accurate to say turn instead of cue here? Seems simpler for non-natives.

```suggestion
New blocks are added to Ethereum every 12 seconds (unless a block proposer misses its turn), so a near-constant stream of data gets added to block explorers. Blocks contain a lot of important data that you may find useful:
```